### ISSUE TITLE
yaml editor in web app could be improved

### ISSUE THREAD

**Issue by jprybylski** [jprybylski/deckifyr#22]
Syntax highlighting would be an easy 1-up. A form-based editor would be even better, similar to what is achieved in the slide editor. A toggle between the form-based and direct syntax highlighted version (which should have validation if it doesn't already) would be ideal
